src/commands/app/init.js

```diff
@@ -18,6 +18,7 @@ const chalk = require('chalk')
 // const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app:init', { provider: 'debug' })
 const { Flags } = require('@oclif/core')
 const generators = require('@adobe/generator-aio-app')
+const hyperlinker = require('hyperlinker')
```

Comment:
since we are already including @oclif/core, why not use `CliUx.ux.url`? So we don't need another dependency.
https://github.com/oclif/core/tree/main/src/cli-ux

Reply:
Hi @shazron ,

- oclif hyperlink functionality `CliUx.ux.url` seems to be broken, It returns promise back
- Unable to use it with template-literals .. It is not working despite many efforts
- And `oclif` has underlying same package `hyperlink` for `CliUx.ux.url` functionalilty.

Thanks,

Reply:
Looks like CliUx.ux.url does not return the "hyperlinked" text, but modifies the text stream directly. Let's not waste any more time on this and use hyperlink, your original plan